app/core/src/main/java/com/fsck/k9/job/K9Job.kt

```diff
@@ -0,0 +1,25 @@
+package com.fsck.k9.job
+
+import com.evernote.android.job.Job
+import com.fsck.k9.Account
+
+interface K9JobAction {
+    fun scheduleJob(account: Account)
+}
+
+abstract class K9Job : Job(), K9JobAction {
+
+    companion object {
+        const val EXTRA_KEY_ACCOUNT_UUID = "param_key_account_uuid"
+    }
+
+    protected fun getSyncIntervalInMillisecondsIfEnabled(account: Account): Long? {
```

Comment:
Sync and push refresh intervals are different. This one is specific to `MailSyncJob`. For `PusherRefreshJob` use `Account.getIdleRefreshMinutes()`.